### PR TITLE
Species tree pipeline improvements

### PR DESCRIPTION
**Pull request by ens-sb**
## Description

_Describe the problem you're addressing here._

**Related JIRA tickets:**
- ENSCOMPARASW-5667
- ENSCOMPARASW-5665

#### Change 1
Mapped BUSCO gene ids to simpler one to avoid GenBlas bug.

#### Change 2
Nextflow config updates and more resource classes in the shared config.

#### Change 3
Added gffread and trimal as separate processes.

#### Change 4
Some minow iqtree tweaks.

## Testing
The pipeline was run on a small artificial dataset.

### CODE REVIEW

File: pipelines/SpeciesTreeFromBusco/main.nf [hunk 1]
```diff
@@ -123,11 +137,30 @@ process alignProt {
...
+    input:
+        path full_aln
+    output:
+        path "trimmed_alignments/trim_aln_*.fas", emit: trim_aln
+
+    script:
+    id = (full_aln =~ /.*prot_(.*)\.fas$/)[0][1]
+    """
+    mkdir -p trimmed_alignments
+    trimal -automated1 -in $full_aln -out trimmed_alignments/trim_${id}.fas
```
[the remark this answers is not in the document]
Comment (ens-sb) [reply pipelines/SpeciesTreeFromBusco/main.nf:156]: Thanks, that is a good idea. Will switch to gappyout.

File: pipelines/nextflow.config [hunk 2]
```diff
@@ -60,18 +60,32 @@ process {
         executor = 'slurm'
         queue = '32c128m60d'
         clusterOptions = ' -n 32 -C0 -M64000 '
+    }
+	withLabel: retry_with_4gb_mem_c1 {
```
[the remark this answers is not in the document]
Comment (ens-sb) [reply pipelines/nextflow.config]: Fixed these to be consistent. Though my nextflow files use tab indent as the vim plugin does not handle indentation.

File: pipelines/SpeciesTreeFromBusco/main.nf [hunk 3]
```diff
@@ -68,13 +69,26 @@ process buscoAnnot {
...
+        path "cdna/*.fas"
+    script:
+    """
     mkdir -p cdna
-    ${params.gffread_exe} -w cdna/$genome -g $genome anno_res/busco_output/annotation.gtf
+    ${params.gffread_exe} -w cdna/$genome -g $genome $busco_annot
     """
 }
 
+
```
[the remark this answers is not in the document]
Comment (ens-sb) [reply pipelines/SpeciesTreeFromBusco/main.nf]: I have added the docstrings for process inputs and outputs.

**Review by CristiGuijarro — APPROVED** [review]
Looks good :+1:

**Pull request merged**